tests/functional/builtins/codegen/test_create_functions.py

```diff
@@ -110,13 +110,17 @@ def test2(a: uint256) -> Bytes[100]:
     assert receipt["gasUsed"] < GAS_SENT
 
 
-def test_create_minimal_proxy_to_create2(get_contract, create2_address_of, keccak, tx_failed):
-    code = """
+@pytest.mark.parametrize("revert_on_failure", [True, False, None])
+def test_create_minimal_proxy_to_create2(
+    get_contract, create2_address_of, keccak, tx_failed, revert_on_failure
+):
+    revert_arg = "" if revert_on_failure is None else f", revert_on_failure={revert_on_failure}"
+    code = f"""
 main: address
 
 @external
 def test(_salt: bytes32) -> address:
-    self.main = create_minimal_proxy_to(self, salt=_salt)
+    self.main = create_minimal_proxy_to(self, salt=_salt{revert_arg})
```

Comment:
i don't think we actually need `revert_on_failure` for the proxy create as i don't think it's really possible to fail (unless there is an EVM bug or not). in any case it shouldn't be configurable by the user, we should always pick one or the other.

Reply:
Well, there is a possibility to have a revert: address collision in the `CREATE2` case.